### DISCUSSION TITLE
Blog comment on 32x Reduced Memory Usage With Binary Quantization

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1898] [Docs/blog comments]
# Blog comment on 32x Reduced Memory Usage With Binary Quantization

In-depth technical breakdown of how binary quantization works and how to use it in Weaviate.

https://weaviate.io/blog/binary-quantization

**Comment by Ayesha-Imr** [comment]
This is one of the best written articles I have ever read. From knowing next to nothing about BQ, I have understood exactly what it is and all the benefits it provides clearly. Kudos to the Weaviate team for their excellent work. Looking forward to putting BQ into use myself!

**Reply by databyjp** [reply]
@Ayesha-Imr thank you for the comment! @zainhas and @abdelr are superstars for sure 🤩
Check out [this guide](https://weaviate.io/developers/weaviate/configuration/bq-compression) to try out BQ!
